features/gezag-minderjarige/4a.2-ouders-overleden-of-onbevoegd-tot-gezag.feature

```diff
@@ -65,10 +65,17 @@ Functionaliteit: 4a.2 - Ouders overleden of onbevoegd tot gezag
   • Mogelijk moet autorisatie worden verleend voor Beschrijving Document vanwege de instructie opnemen “van rechtswege, erkenning OV 2022”.
 
   Gebruikte velden:
+    - Burgerservicenummer ouder 1 -> 02.01.20
+    - Geslachtsnaam ouder 1 -> 02.02.40
+    - Burgerservicenummer ouder 1 -> 03.01.20
+    - Geslachtsnaam ouder 1 -> 03.02.40
+    - Burgerservicenummer huwelijk/geregistreerd partnerschap -> 05.01.20
+    - Start datum huwelijk/geregistreerd partnerschap -> 05.06.10
```

Comment:
in deze  feature (/ deze stap in de gezagsmodule) worden geen huwelijk of partnerschap velden gebruikt

Reply:
Als je deze usage pad volgt:

```
HuwelijkOfPartnerschap::getDatumVoltrokken
-> gebruikt in Persoonslijst::checkHopRelaties
-> gebruikt in GezagService::ophalenOuder1
-> gebruikt in GezagBepaling::getPlOuder1
-> gebruikt in vragen 4a.2, 4a.3, 4b.1
```

zou je als het goed is ook op dezelfde conclusie komen.